src/pages/List/List.js

```diff
@@ -1,7 +1,173 @@
+      .then(result => {
+        setItems(result);
+      });
+  };
+
+  const moreItem = e => {
+    e.preventDefault();
+    fetch(
+      `http://10.58.7.24:8000/posts?title_or_hashtag=${title_or_hashtag}&city=${city}&categories=${categories}&offset=${
+        Result + 8
+      }`
+    )
+      .then(res => res.json())
+      .then(result => {
+        setItems(result);
+      });
+  };
 
+  const handleInput = e => {
+    const { name, value } = e.target;
+    setUser({ ...user, [name]: value });
+  };
+  const onCheckedElement = (checked, item) => {
+    if (checked) {
+      setUser({ ...user, categories: categories.concat(item, ',') });
+    } else if (!checked) {
+      setUser({
+        ...user,
+        categories: categories.replace(`${item},`, ''),
+      });
+      // setUser({ ...user, categories: categories.filter(el => el !== item) });
+    }
+  };
+  // 검색 기능
+
+  // 지도를 표시할 div와  지도 옵션으로  지도를 생성합니다
+
+  // const load_more = event => {
+  //   setPaginate(prevValue => prevValue + 5);
+  // };
+  return (
+    <div>
+      {Object.keys(items).length !== 0 && (
+        <S.Inner>
+          <S.Wrapper>
+            <S.Header>
+              <S.HeaderTitle>새싹 여행</S.HeaderTitle>
+              <S.HeaderSubTitle>
+                새싹과 관련된 지역별 추천장소를 소개합니다
+              </S.HeaderSubTitle>
+            </S.Header>
+            <S.RightSide>
+              <S.RightWrapper>
+                <S.LeftSide>
+                  <KakaoMap data={data} />
+                </S.LeftSide>
+                <S.RightTitle>추천 장소</S.RightTitle>
+                <S.RecommendPlace>
+                  <RecommendList RecommendPostList={RecommendPostList} />
+                </S.RecommendPlace>
+                <S.SearchContainer>
+                  <S.Region name="city" onChange={handleInput}>
+                    <option value="">도시</option>
+                    {COUNTY.map(item => (
+                      <option key={item.id} value={item.county}>
+                        {item.county}
+                      </option>
+                    ))}
+                  </S.Region>
+                  <S.SearchBox
+                    type="search"
+                    name="title_or_hashtag"
+                    id="search-form"
+                    placeholder="장소명 혹은 해시태그를 검색하세요"
+                    onChange={handleInput}
+                  />
+                </S.SearchContainer>
+                <S.CheckBox>
+                  {CATEGORY_LIST.map(item => {
+                    return (
+                      <S.CheckBoxTo key={item.id}>
+                        <S.CheckBoxType
+                          onChange={e => {
+                            onCheckedElement(e.target.checked, e.target.value);
+                          }}
+                          type="checkbox"
+                          value={item.data}
+                          name="categories"
+                          checked={categories.includes(item.data) && true}
```

Comment:
includes 메서드 자체가 boolean type을 return하기 때문에 이렇게만 쓰셔도 됩니다.
```suggestion
                          checked={categories.includes(item.data)}
```